xbmc/utils/LangCodeExpander.cpp

```diff
@@ -921,6 +921,7 @@ const std::array<struct LCENTRY, 540> g_iso639_2 = {{
     {MAKECODE('\0', 'd', 'e', 'l'), "Delaware"},
     {MAKECODE('\0', 'd', 'i', 'n'), "Dinka"},
     {MAKECODE('\0', 'd', 'i', 'v'), "Divehi"},
+    {MAKECODE('\0', 'd', 'i', 'v'), "Dhivehi"},
```

Comment:
This code is identical to the line above? Are dictionary entries not supposed to be unique in this map?

Reply:
some english names in the ISO 639-2 can have name variations with same code, another example is "Haitian" that have 2 names variants
it is possible that others multiple names are missing (i have not check its out of scope of this PR) i found this one by chance,
despite this anyway the root of problem (mentioned on PR "sidenote") will be fixed by #26011